ccd-definition/AuthorisationCaseField/CareSupervision/cafcass.json

```diff
@@ -1594,5 +1594,47 @@
     "CaseFieldID": "previousProceedingListLA",
     "UserRole": "caseworker-publiclaw-cafcass",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "carePlanList",
+    "UserRole": "caseworker-publiclaw-cafcass",
+    "CRUD": "CRUD"
```

Comment:
wonder why cafcass need "CUD"? Think they can't upload any document, "R" should be enough for them?

Reply:
I don't know. I copy it from `applicationDocuments`